app/views/admin/companies/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_for [:admin, @company] do |f| %>
+  <%= render "partials/signup_form", f: f  %>
```

Comment:
^ Here is what I added to make use of the partial on both the :edit and :new views. Since the entire form isn't reused, I only left the duplicate part on the partial, and just included the unique stuff in each of the respective views.